src/requirements/data/university/index.ts

```diff
@@ -1,35 +1,27 @@
-import { BaseRequirement } from '../../types';
+import { Course, CollegeOrMajorRequirement } from '../../types';
+import { includesWithSingleRequirement } from '../checkers-common';
 
-const universityRequirements: readonly BaseRequirement[] = [
-  {
-    name: 'Academic Credits',
-    description: 'To graduate, a student must earn a minimum of 120 academic credits. '
-      + 'Physical education credits and “10XX” courses do not count toward the 120 required credits.',
-    source: 'http://courses.cornell.edu/content.php?catoid=31&navoid=7901',
-    operator: 'or',
-    fulfilledBy: 'credits',
-    minCount: 120,
-    applies: 'all',
-    progressBar: true
-  },
+const universityRequirements: readonly CollegeOrMajorRequirement[] = [
   {
     name: 'Physical Education',
     description: 'All incoming freshmen are required to take two credits (two courses) of Physical Education, '
       + 'one credit each semester of the first year on campus.',
-    source: 'http://courses.cornell.edu/content.php?catoid=36&navoid=9249',
+    source: 'http://courses.cornell.edu/content.php?catoid=41&navoid=11637',
+    checker: (course: Course): boolean => 'PE'.includes(course.subject),
```

Comment:
Should the checker just be checking PE courses, or also the 10XX limit

Reply:
just PE bc this is the PE requirement

Reply:
Wow it is 🤦  oops thank you